README.md

```diff
@@ -171,7 +180,6 @@ the `DefaultKubeConfig` constant in the `giantswarm/e2e-harness/pkg/harness` pac
 - https://github.com/giantswarm/kubernetes-node-health
 - https://github.com/giantswarm/kvm-operator
 - https://github.com/giantswarm/net-exporter
-- https://github.com/giantswarm/operatorkit
 - https://github.com/giantswarm/release-operator
```

Comment:
I just ran into issues where I need to do vendor updates in order. So I thought splitting projects into categories makes sense here.